apps/web/src/app/admin-console/organizations/tools/import/org-import-routing.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from "@angular/core";
+import { RouterModule, Routes } from "@angular/router";
+
+import { Organization } from "@bitwarden/common/admin-console/models/domain/organization";
+
+import { OrganizationPermissionsGuard } from "../../guards/org-permissions.guard";
+
+import { OrganizationImportComponent } from "./org-import.component";
+
+const routes: Routes = [
+  {
+    path: "import",
```

Comment:
`import` is also in the path from the parent routing module, so currently this requires you to navigate to `tools/import/import`.

```suggestion
    path: "",
```